README.md

```diff
@@ -11,7 +11,7 @@ Clone the ddc-schemas repo:
 
 Checkout a particular version of the schema:
 
-    cd ddc-schemas && git checkout storage-v0.1.2
+    cd ddc-schemas && git checkout storage-v0.1.3
```

Comment:
You can create a new tag in ddc-schemas, and then commit it. See below:

```
Freeze the schema version and generated code

    git add ddc-schemas model/pb
```

Reply:
The command with the version was an example, we could rewrite it as:

    cd ddc-schemas && git checkout storage-vX.X.X